pkg/api/v1alpha1/router.go

```diff
@@ -438,6 +468,80 @@ func (r *Router) Routes(rg *gin.RouterGroup) {
 		r.AuthMW.AuthRequired(readScopesWithOpenID("governor:applications")),
 		r.listApplicationTypeApps,
 	)
+
+	rg.GET(
+		"/notification-types",
+		r.AuditMW.AuditWithType("ListNotificationTypes"),
+		r.AuthMW.AuthRequired(readScopesWithOpenID("governor:notifications")),
```

Comment:
In general, I wonder if `notification-types` and `notification-targets` deserve their own scope types? thoughts?

Reply:
I would go for simplicity and keep it as is until we have a requirement for having two scopes